inc/compatibilities/master_slider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Class Optml_master_slider.
+ *
+ * @reason Added classes to watch for background lazyload
+ */
+class Optml_master_slider extends Optml_compatibility {
+
+
+
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		return is_plugin_active( 'master-slider/master-slider.php' );
+	}
+
+	/**
+	 * Register integration details.
+	 */
+	public function register() {
+		add_filter(
+			'optml_lazyload_bg_selectors',
+			function ( $all_watchers ) {
+				$all_watchers = array_merge( $all_watchers, ['.master-slider'] );
+				return $all_watchers;
+			}
+		);
+		$settings = new Optml_Settings();
+
+		if ( $settings->get( 'lazyload' ) === 'enabled' ) {
+			add_filter(
+				'optml_lazyload_filename_excluded',
+				function ( $array ) {
+					$array['blank.gif'] = true;
```

Comment:
here I think it's better to exclude at all the optimization on this, not just the lazyload.

Reply:
ok